requirements/pip.txt

```diff
@@ -18,9 +18,12 @@ django-vanilla-views==1.0.4
 pytest-django==2.8.0
 
 requests==2.3.0
-slumber==0.6.0
+slumber==0.6
```

Comment:
Is there a more specific version to target for py3 support? This can still target `0.6.0`